assets/react/v3/shared/config/styles.ts

```diff
@@ -215,6 +215,13 @@ export const colorTokens = {
       onHold: '#ac0640',
       cancelled: '#6f7073',
     },
+    alert: {
+      warning: '#D47E00',
+      success: '#D47E00', // will be updated later
+      danger: '#D47E00', // will be updated later
+      info: '#D47E00', // will be updated later
+      primary: '#D47E00', // will be updated later
+    }
```

Comment:
This pattern does not align with our design system. If it is out of our design system then move these colors to the Alert component and define there.